omnn/math/CMakeLists.txt

```diff
@@ -28,18 +30,18 @@ if(NOT WIN32)
 	set(DEPENDENCIES ${DEPENDENCIES} boost_thread boost_serialization boost_system)
 endif()
 
-lib(${DEPENDENCIES})
```

Comment:
this lib function declares new library project with dependencies enumerated in params (${DEPENDENCIES})